gp-templates/translations.php

```diff
@@ -290,7 +245,11 @@
 	endif;
 ?>
 </table>
-<?php echo gp_pagination( $page, $per_page, $total_translations_count ); ?>
+<?php
+gp_translations_bulk_actions_toolbar( $bulk_action, $can_write, 'bottom' );
+
+echo gp_pagination( $page, $per_page, $total_translations_count ); // WPCS: XSS Ok.
```

Comment:
`// WPCS: XSS Ok.` shouldn't be necessary anymore since it's now part of `customAutoEscapedFunctions`.

Reply:
Removed.